S21/harisalam/lowerCase.py

```diff
@@ -0,0 +1,46 @@
+import pandas as pd
+import re
+
+
+def lowerCasing(word: str, lan: str):
+
+    if (re.search('^en.*', lan)):
+        return word.lower()
+
+    if (re.search('^tr.*', lan) or re.search('^az.*', lan)):
+        replace = word.replace('I', u"\u0131")
+        # print(line)
+        return replace.lower()
+
+    if (re.search('^ga.*', lan)):
+        vowels = 'A,E,I,O,U,Á,É,Í,Ó,Ú'.split(',')
+        if ((re.search('^[n,t].*', word)) and (word[1] in vowels)):
+            return((word[0]+'-'+word[1:]).lower())
+        else:
+            return(word.lower())
+
+    if (re.search('^el.*', lan)):
+        if(re.search('Σ$', word)):
```

Comment:
Test failed for word 'ΠΌΛΣΗΣ' the result is 'πόλςης' where the expected result was 'πόλης'Test failed for word 'ΠΌΛΣΗΣ' 
This will not work if sigma is present in the end as well as the somewhere in the middle